RLBotPack/Diablo/setup.py

```diff
@@ -0,0 +1,33 @@
+import urllib.request
+import zipfile
+
+
+def downloadFile():
+    try:
+        urllib.request.urlretrieve("https://urlzs.com/XSvMA", 'rlutilities.zip')
+    except Exception as e:
+        print("Error downloading file!")
+        print(e)
+        _ = input("Press enter to exit")
+        quit()
+
+def unzipFile():
+    try:
+        with zipfile.ZipFile("rlutilities.zip","r") as zip_ref:
+            for _file in zip_ref.namelist():
+                zip_ref.extract(_file)
+
+    except Exception as e:
+        print("Error unzipping the file!")
+        print(e)
+        _ = input("Press enter to exit")
+        quit()
+
+if __name__ == "__main__":
```

Comment:
The RLBotGUI will not be executing your setup.py, and instead will rely on a solution for retrieving git-lfs files. Since the rlutilities folder will still be inside Diablo after this PR, I think there's no problem - https://github.com/RLBot/RLBotPack/tree/master/RLBotPack/Diablo/rlutilities.

Just wanted to call your attention to this in case you're concerned.

Reply:
no worries